subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -150,83 +94,26 @@ static bool compare(const char *s1, const char *s2)
 	return !strncmp(s1, s2, strlen(s2));
 }
 
-int nrf_codec_init(void)
+static void nrf_cloud_decode_desired_obj(cJSON *const root_obj,
```

Comment:
nit: simply `nrf_cloud_decode_obj`, or do we have that already?

Reply:
this was to combine some duplicate code which gets the json object with the key "desired".